random_message.py

```diff
@@ -1,7 +1,7 @@
 import datetime
 import random
 
-weekday = ["Monday 😛",
+weekdays: list[str] = ["Monday 😛",
            "Tuesday 🍒",
            "Wednesday 😬",
            "Thursday 🥱",
```

Comment:
**Suggestion:** Use the typing.List instead of list for type hinting.
```suggestion
from typing import List

weekdays: List[str] = ["Monday 😛",
            "Tuesday 🍒",
            "Wednesday 😬",
            "Thursday 🥱",
```